azerothcore/mod-anticheat
# Upgrade module

## Changes Proposed:
-  Temporary patch, for future people who use the module, so that they do not have to load the SQL manually.
-  Until the module detects the base files first, and then updates them, this is the only way the module will work when run for the first time.

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Windows 10
- In-game

## How to Test the Changes:


1. Add the module.
2. Enter with a character to the kingdom.
3. The server no longer crashes, because the SQL is up to date. You don't have to do it manually.
4. Without this change, you are forced to apply the changes manually. Because if not, the emulator detects that the tables are not updated.
5. Maybe it is not the best solution, but at least it is a solution for future users. Those who already have the module should sacrifice the tables or continue updating it manually.

## Known Issues and TODO List:


## Screenshot

![no_crash_report](https://github.com/azerothcore/mod-anticheat/assets/2810187/aa857ad7-96f6-4d5f-a417-a78e79ac7e3d)


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
